tests/TestCase/ORM/QueryTest.php

```diff
@@ -1841,6 +1841,94 @@ public function testInsert()
         $this->assertEquals(2, $result->rowCount());
     }
 
+    /**
+     * Test insertAll method inserts more than one record
+     *
+     * @return void
+     */
+    public function testInsertAll()
+    {
+        $table = $this->getTableLocator()->get('articles');
+
+        $result = $table->query()
+            ->insertAll(
+                ['title'],
+                [
+                    ['title' => 'same ol same'],
```

Comment:
There should be a test with non string values too showing the that type map set from table is used.